### DISCUSSION TITLE
Can you snap a photo of the back of the proto board as well?

### DISCUSSION THREAD

**Discussion by drudru** [markadev/AppleII-VGA#10] [General]
Hi @markadev 

I like how you prototyped the board.
Can you also show a picture of the back of the prototype board as well?
I am curious as to how you wired it up.

**Comment by markadev** [comment]
Yeah sure.  There's not much on the back - mainly just power & ground I think.  The wire sticking down with blue tape on it is just a wire I left on to easily probe the bus clock directly from the bus connector.

![board_front](https://github.com/markadev/AppleII-VGA/assets/17986295/2b450ed6-1a25-402c-b692-1eabdf0e2da5)
![board_rear](https://github.com/markadev/AppleII-VGA/assets/17986295/e67f8ec2-0fa1-49f5-a547-65ccd1773e37)

**Reply by drudru** [reply]
This is really helpful. Yeah - not much for the back which I think is compelling.

Thank you
